spec/models/authority_spec.rb

```diff
@@ -1,6 +1,16 @@
 require 'spec_helper'
 
 describe Authority do
+  describe "validations" do
+    it 'should ensure a unique short_name' do
+      @a1 = create(:authority, short_name: "Existing")
+      @a2 = build(:authority, short_name: "Existing")
```

Comment:
The only thing to remember with that is there can be trade off between DRYness and how clear an individual test is. That's why helpful variable names are important I guess ;-) If you have to look in a bunch of places to understand what a test is basically doing then I think it's worth considering inlining the variables, or improve the naming.